examples/counter-next/shared/.gitignore

```diff
@@ -0,0 +1,4 @@
+!/generated
```

Comment:
Are we intentionally comitting the generated code? 🤔

Reply:
as per our convo, they're no longer comitted